src/core.js

```diff
@@ -91,6 +91,8 @@ const patchFactory = function(run) {
     doc = node.ownerDocument;
     currentParent = node.parentNode;
 
+    importNode(node);
```

Comment:
Ensures everything under `node` has a NodeData. Eventually I want to remove the if conditional from `getData`.